### DISCUSSION TITLE
Possible to turn on armed away using HA

### DISCUSSION THREAD

**Discussion by pwnypower** [petrleocompel/hikaxpro_hacs#88] [Q&A] [answered]
Hey Petrle,

Great to see an integration like this. I am looking into an AX Pro system myself but i would like to know if it's possible to turn on or off the alarm using this integration. in my old setup(verisure) i used my location to arm or disarm the system along with some other options. 

is this possible with this integration?

kind regards,
Tim

**Comment by petrleocompel** [comment, marked as answer]
Yes it is. Integration can arm/disarm. Automation how is up to you.
